sphericart-jax/setup.py

```diff
@@ -1,6 +1,7 @@
 import os
 import subprocess
 import sys
+import multiprocessing
```

Comment:
```suggestion
```

Reply:
Hmmm thanks, it seems that the linter didn't catch that, I will investigate